iocell.8.txt

```diff
@@ -1196,6 +1196,17 @@ PROPERTIES
     Default: off
     Source: cpuset(1)
 
+  volumes=/host/dir[:/jail/dir[:mount_options]]
+
+  Mounts and unmounts directories from the host into the jail root using
+  nullfs(5).
+  'mount_options' are passed direcly to the mount command and defaults to "rw"
+  if ommitted.
+  If '/jail/dir' is also ommitted then the same value as '/host/dir' will be
```

Comment:
I'd drop any configuration modes that can be done in more than one way, to avoid confusions and make automation easier, following Python's matra of 'there should be one and only one way to do it'.